### PR TITLE
add PR review bot

### PR DESCRIPTION
**Pull request by rootfs**
The secrets are added to the repo

image

### CODE REVIEW

**Comment by rootfs** [comment]
@wenboown can you take a look? Thank

**Comment by rootfs** [comment]
@wenboown can you take a look? thanks

File: .github/workflows/review_bot.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+#      - name: Dump GitHub context for debug
+#        env:
+#          GITHUB_CONTEXT: ${{ toJson(github) }}
+#        run: |
+#          echo "$GITHUB_CONTEXT"
+      - name: Dump event payload for debug
+        run: cat $GITHUB_EVENT_PATH
+      - name: Run SeineSailor
+        uses: SeineAI/SeineSailor@dev-action-only-do-not-merge-or-delete
```
Comment (wenboown) [inline .github/workflows/review_bot.yml:81]: use SeineAI/SeineSailor@main here.

Also, I think this PR needs to be merged first for it to work. So next PR should be able to trigger it.

**Review by wenboown — APPROVED** [review]

File: .github/workflows/review_bot.yml (hunk 1 above)
Comment (rootfs) [reply .github/workflows/review_bot.yml:81]: thanks!

**Pull request merged**